modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -488,8 +488,16 @@ class CV_EXPORTS BufferPool
     //! Allocates a new GpuMat of given size and type.
     GpuMat getBuffer(int rows, int cols, int type);
 
+// WARNING: unreachable code using Ninja
+#if defined _MSC_VER && _MSC_VER >= 1928
```

Comment:
> 1928

Just 1920 (MSVS 2019 base version) as we probably have we same problem with 1927 etc.

https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

Reply:
I just wanted to pin a version which caught a warning and I didn't test it on 1920, but updated changes according to your comment.